lib/grape/validations/attributes_iterator.rb

```diff
@@ -5,11 +5,12 @@ class AttributesIterator
 
       attr_reader :scope
 
-      def initialize(validator, scope, params)
+      def initialize(validator, scope, params, multiple_params: false)
```

Comment:
How about writing another attribute iterator, `MutlipleParamsAttributeIterator` instead of this additional parameter?

Reply:
Great idea, thanks! I changed this part, please check it out. Also the build is green now.